tests/test_helper/setup_teardown/bgp.bash

```diff
@@ -4,6 +4,8 @@ setup_file() {
     load test_helper/microovn.bash
     load test_helper/bgp_utils.bash
 
+    # Ensure that required VRF kernel module is available on the host
+    lsmod | awk '{print $1}'| grep -i "\<vrf\>"
```

Comment:
See above comment on using LXD profile to load the module.

Reply:
Thank you, yes, that was indeed better approach. This was removed.